specification/storagecache/resource-manager/Microsoft.StorageCache/stable/2021-03-01/storagecache.json

```diff
@@ -554,18 +514,66 @@
         "operationId": "Caches_Update"
       }
     },
+    "/subscriptions/{subscriptionId}/resourcegroups/{resourceGroupName}/providers/Microsoft.StorageCache/caches/{cacheName}/storageTargets/{storageTargetName}/dnsRefresh": {
```

Comment:
nit: POST action names are generally verbs. IMO, name 'refreshDns' is apt here. I will leave the call to you.

Reply:
I discussed with others on the team and they requested that I keep it as is, so I did not update it here.